Selection Principles update

ccaruvana opened the pull request:
This incorporates all changes made in https://github.com/pi-base/data/pull/328 and partially addresses https://github.com/pi-base/data/issues/327.

> Updates to https://topology.pi-base.org/spaces/S000022.
> Updated https://topology.pi-base.org/theorems/T000141 to sigma-compact implies omega-Menger. (Note that T362 asserts that omega-Menger implies Menger, so nothing is lost in this change.)
> Added T363 to deduce that S22 is not Markov omega-Menger.
> Added a topological group S179 that is Lindelof but not an epsilon-space.

prabau commented:
I have not looked at selection principles before, so here is a basic question.

The notion of $\omega$-cover was introduced by Gerlits and Nagy in https://www.sciencedirect.com/science/article/pii/0166864182900657 with the definition that you have.  Scheepers in https://www.sciencedirect.com/science/article/pii/0166864195000674 defines it with the additional condition that no element of the cover contains $X$ (which seems the same as saying $X$ does not belong to the cover?).  Does this difference matter?

ccaruvana commented:
Yeah, the latter is the most common definition. I'm not totally sure off the top of my head what kind of trivialities in terms of the selection principles can arise that are really "problematic." I went with the current formulation since that's what's analogous to https://topology.pi-base.org/properties/P000128, but I will make the changes for non-trivial covers, since that's what's most common in the literature now.

prabau commented:
For T362 (omega-Menger ==> Menger), is it really clear?  Why does $\mathsf S_{\mathrm{fin}}(\Omega,\Omega)$ imply $S_{fin}(\mathcal{O},\mathcal{O})$, since an open cover is not an $\omega$-cover in general?  What am I missing?

prabau commented:
> Yeah, the latter is the most common definition. I'm not totally sure off the top of my head what kind of trivialities in terms of the selection principles can arise that are really "problematic." I went with the current formulation since that's what's analogous to https://topology.pi-base.org/properties/P000128, but I will make the changes for non-trivial covers, since that's what's most common in the literature now.

I would be good to understand.  If a cover contains $X$, isn't it automatically an $\omega$-cover?  And in that case, by choosing $X$ in the selection, one generates something that is also an $\omega$-cover, so it's all trivial.  So wouldn't the definitions of $\omega$-Menger, etc be the same with or without the extra condition?

ccaruvana commented:
> For T362 (omega-Menger ==> Menger), is it really clear? Why does Sfin(Ω,Ω) imply Sfin(O,O), since an open cover is not an ω-cover in general? What am I missing?

I think here all you have to do is close your open cover under finite unions and apply the $\Omega$ selection principle. Since a finite selection of finite unions can be unraveled into a finite selection, that settles it. The single selection case is not so clear and is a result of Sakai, the paper cited for that particular definition. If you want, I can be more clear and say "by Theorem x" in the definition.

ccaruvana commented:
> > Yeah, the latter is the most common definition. I'm not totally sure off the top of my head what kind of trivialities in terms of the selection principles can arise that are really "problematic." I went with the current formulation since that's what's analogous to https://topology.pi-base.org/properties/P000128, but I will make the changes for non-trivial covers, since that's what's most common in the literature now.
> 
> I would be good to understand. If a cover contains X, isn't it automatically an ω-cover? And in that case, by choosing X in the selection, one generates something that is also an ω-cover, so it's all trivial. So wouldn't the definitions of ω-Menger, etc be the same with or without the extra condition?

You're definition right on that, but I think some combinatorial things come up. If you assume that $X$ isn't in the cover, then the covers need be "large," namely, that any finite set is contained in infinitely many distinct open sets of the $\omega$-cover. This is sometimes used and is clearly not necessarily the case if your cover is just {X}, or any finite cover of the space for that matter.

prabau commented:
> I can be more clear and say "by Theorem x" in the definition.

I think that would be helpful.

ccaruvana commented:
Okay, I think I've addressed all of the covers being so that $X$ itself isn't in them and also updated the $\omega$-blank properties with references to page numbers/numbered theorems.

ccaruvana commented:
Oh, I updated the results in the properties themselves. E.g. $\omega$-Rothberger is equivalent to the space being Rothberger in each of its finite powers. Therefore, an $\omega$-Rothberger space is Rothberger, using that definition. Should I copy the lines from the properties to the theorem statements?

prabau commented:
> Oh, I updated the results in the properties themselves. E.g. ω-Rothberger is equivalent to the space being Rothberger in each of its finite powers. Therefore, an ω-Rothberger space is Rothberger, using that definition. Should I copy the lines from the properties to the theorem statements?

Either way, but it's probably ok the way it is.  With this equivalence, the implication $\omega$-Menger ==> Menger, etc is indeed evident.

Btw, for $\epsilon$-space, shouldn't the definition be that $\omega$-covers have countable $\omega$-subcovers (instead of finite)?

ccaruvana commented:
> Btw, for ϵ-space, shouldn't the definition be that ω-covers have countable ω-subcovers (instead of finite)?

Yes, of course. I fixed that.

prabau commented on properties/P000149.md:
```diff
@@ -0,0 +1,13 @@
+---
+A family $\mathcal U$ of open subsets of a topological space $X$ is called an $\omega$-cover if every finite subset of $X$ is contained in one of the members and $X \not\in \mathcal U$.
+
+A space $X$ is an $\varepsilon$-space if every open $\omega$-cover has a countable $\omega$-subcover. Equivalently, a space $X$ is a $\varepsilon$-space if every finite power of $X$ is
```
the word "open" is redundant given the def of $\omega$-cover in the previous sentence.

ccaruvana replied:
Addressing these now.

prabau commented on properties/P000150.md:
```diff
@@ -0,0 +1,14 @@
+uid: P000150
+name: $\omega$-Rothberger
+refs:
+- doi: 10.2307/2046816
```
10.1090/S0002-9939-1988-0964873-0 seems a better doi (open access from AMS website)

prabau commented:
What does the settings.json file do?

ccaruvana commented:
> What does the settings.json file do?

Actually, I was wondering that myself. It loaded up as soon as I opened the branch. I'm not sure if it has something do with how I was interfacing, but I'm sure it can be deleted.

prabau commented:
Well, that's all for my comments.  Looks pretty good from what I can see, but I know nothing about games and have not looked at that part.  Hope Lynn or Steven can take a look and approve.

One thing I was curious though.  Comparing P128 ($k$-Lindelöf) and P149 ($\varepsilon$-space = $\omega$-Lindelöf), they both have something with "Lindelöf" in one of their names, but the former has finite subcovers and the latter has countable subcovers.  No consistency there, and I don't have access to the paper for P128, but maybe you can confirm?

ccaruvana commented:
> One thing I was curious though. Comparing P128 (k-Lindelöf) and P149 (ε-space = ω-Lindelöf), they both have something with "Lindelöf" in one of their names, but the former has finite subcovers and the latter has countable subcovers. No consistency there, and I don't have access to the paper for P128, but maybe you can confirm?

Ah, so that's where my mistake in $\varepsilon$-space originated from. I had copy-pasted from $k$-Lindelof. So I don't have access to the referenced articles (apparently it appears in a book), but taking a look at https://arxiv.org/abs/1201.1576 corroborates the suspicion that it should be countable. I've always seen $k$-Lindelof defined as having countable sub-$k$-covers, so I've made that chance. Perhaps we can update this reference in the future with something a little more accessible.

prabau commented on spaces/S000043/properties/P000149.md:
```diff
@@ -0,0 +1,10 @@
+space: S000043
+property: P000149
+value: false
+refs:
+  - mr: MR3728284
+    name: Topology, 2nd Edition (Munkres)
+---
+
+As proved on page 193 of {{mr:MR3728284}}, the {S76} is not {P18}, so the Sorgenfrey line is not Lindelöf in all of its finite powers.
```
For S43/P149, the justification that S76 is P18 does not need to be justified by Munkres, I would say, as the user can just click S76 and check it there.  If we need a token something in the refs list, maybe just put Countexamples there?

ccaruvana replied:
Yes, I wanted to simply reference the property for the space, but wasn't sure if there was a way to link directly to the property for the space, particularly. I included Munkres to be on the safe side in terms of references, even if it is a bit overkill. I will be happy to update this. For S76/P18, we currently mention that the antidiagonal is closed and discrete and mention that every closed subset of a Lindelof space is Lindelof. That's effectively what's in Munkres, as well, and I would imagine what's in Counterexamples.

prabau replied:
created https://github.com/pi-base/web/issues/66

prabau commented on theorems/T000364.md:
```diff
@@ -0,0 +1,14 @@
+
+Mentioned in the proof of Proposition 5 in {{doi:10.1016/j.topol.2005.07.015}}.
+
+Given any cover $\mathscr U$ of a $k$-Lindelöf space $X$, consider $\mathscr V = \left\{ \bigcup \mathscr F : \mathscr F \in [\mathscr U]^{<\omega} \right\}$ where $[\mathscr U]^{<\omega}$ denotes the set of finite subsets of $\mathscr U$. Then $\mathscr V$ is a $k$-cover of $X$ and we can find a countable subset which is a $k$-cover of $X$. Notice that such a subset corresponds to a countable su ...
```
"Given any cover" --> "Given any open cover".
At the end, possibly "corresponds to a countable subset of $\mathscr U$ which is a cover of $X$." --> "corresponds to a countable subcover of $\mathscr U$"

ccaruvana replied:
Changed.

prabau commented:
One basic observation.  In a finite set $X$ there is no $\omega$-cover $\mathcal U$, since $X$ finite would have to belong to $\mathcal U$, but $ \mathcal U$ cannot contain $X$.  So finite sets are vacuously $\omega$-Rothberger and all the other $\omega$ properties.  And that matches the characterization of $\omega$-Rothberger in terms of finite powers of $X$ being Rothberger.  (Of course, for finite sets it's not interesting).

Now for countable, there is https://topology.pi-base.org/theorems/T000341 [countable ==> Rothberger].  So by finite power of countable is countable, we would have [countable ==> $\omega$-Rothberger].  (or not hard to prove also from the other definition).  And presumably countable would imply even stronger properties, up to P152 (Markov $\omega$-Rothberger)?  In any case, should T341 be updated?

And similarly, there are no $k$-covers on a compact space.  So any compact space would be $k$-Rothberger or stronger?

prabau commented:
> Yes, that would be a sensible thing to do. On this note, I did want to write T141 as "sigma-compact -> Markov omega-Menger", but since that's not the statement in the paper, I didn't want to go too far off the reference list. However, the proof sketched does show that it's a Markov strategy. Would it be acceptable to strengthen T141 this way with the same proof sketch?

Not sure if you mean TR141 or T341?  In any case, for such a situation I think it's perfectly fine to generalize.  We can say something like the paper proves it in the case of ... but the same argument works for .....

ccaruvana commented:
> > Yes, that would be a sensible thing to do. On this note, I did want to write T141 as "sigma-compact -> Markov omega-Menger", but since that's not the statement in the paper, I didn't want to go too far off the reference list. However, the proof sketched does show that it's a Markov strategy. Would it be acceptable to strengthen T141 this way with the same proof sketch?
> 
> Not sure if you mean TR141 or T341? In any case, for such a situation I think it's perfectly fine to generalize. We can say something like the paper proves it in the case of ... but the same argument works for .....

I will update both T141 and T341. T141 will be "sigma-compact -> Markov omega-Menger" and T341 will  be "countable -> Markov omega-Rothberger".

ccaruvana commented on theorems/T000369.md:
```diff
@@ -0,0 +1,12 @@
+  name: Selection games and the Vietoris space
+---
+
+Evident from the definitions.
```
Added.

ccaruvana commented:
The search for an $\omega$-Lindelof space which is not $k$-Lindelof continues. As noted at https://math.stackexchange.com/questions/4717613, all separable metrizable spaces are $k$-Lindelof, which I've added here as T376.

prabau commented:
By the argument used for the Fortissimo space (S22), isn't it the case that for anticompact spaces the $\omega$-properties are equivalent to the corresponding $k$-properties? (omega-covers are the same as k-covers).  So one could replace the trait with a theorem.  There would actually be a bunch of such theorems: [$\varepsilon$-space + anticompact ==> $k$-Lindelof], etc, etc At least add the one needed for S22, I'd say.  Kind of tedious to add them all though.  Not sure if we want to.  There is no short-circuit for it, right?

prabau commented:
I am not familiar with the Vietoris topology.  What do you recommend as a general reference?  And in particular for the fact that it's also separable metrizable in the case in question.

ccaruvana commented:
> I am not familiar with the Vietoris topology. What do you recommend as a general reference? And in particular for the fact that it's also separable metrizable in the case in question (may want to add a reference to that in T376).

Yes, that's an excellent question. I'll have to find a good reference other than Michael's paper https://www.ams.org/journals/tran/1951-071-01/S0002-9947-1951-0042109-4/S0002-9947-1951-0042109-4.pdf. In the meantime, there is https://math.stackexchange.com/questions/791334 which basically shows the metrizability criterion (the metric space need not be compact for this to work since defining it for the compact sets only still allows you to define the Hausdorff metric). Separability follows from the separability of the "ground" space since you can fit finite subsets of a dense set into any Vietoris neighborhood. I could have sworn these were in Engelking, but I'm only finding questions about continuous mappings and the Vietoris space in there. It is probably in Kechris' Classical Descriptive Set Theory. I'll check that and find an MR number in the relatively near future if it has the results in question.

ccaruvana commented:
> By the argument used for the Fortissimo space (S22), isn't it the case that for anticompact spaces the ω-properties are equivalent to the corresponding k-properties? (omega-covers are the same as k-covers). So one could replace the trait with a theorem. There would actually be a bunch of such theorems: (ε-space + anticompact ==> k-Lindelof), etc, etc At least add the one needed for S22, I'd say. Kind of tedious to add them all though. Not sure if we want to. There is no short-circuit for it, right?

Ah, that's a good point! I had wondered about that and forgot about the anticompact property. I'll make that edit along with the Vietoris reference(s). It would be nice to have a sort of schema generator function to apply such a "X + Y -> f(X)" theorem family...

ccaruvana commented:
For future reference, Theorem 4.22 in Kechris' Classical Descriptive Set Theory states "If X is a metrizable space, so is K(X). If X is separable, so is K(X)." I'll add this reference in along with an MR number as soon as I can visit my office and have access to MathSciNet.

ccaruvana commented:
Also, @prabau since I live my life in a land where all spaces are $T_2$, I get weary around lower separation axioms. I think T374 here might not need $T_1$. The reason I included that hypothesis is because most of the papers I read include a clause like "suppose all spaces are Hausdorff," but also because the proof referenced says basically the following: assume X is not locally compact (but they just need to avoid having a compact neighborhood, so this is weakly locally compact as we currently have) and first-countable. Given a compact K and a neighborhood U of x, $U \setminus (K \cup \{x\}) \neq \emptyset$ so there is some $y \in U \setminus (K \cup \{ x \})$. Now, they say you can find an open set that contains $K \cup \{x\}$ that does not contain $y$. Assuming $T_1$, just take $X \setminus \{y\}$, but maybe you can just take $X \setminus \overline{\{y\}}$. However, I'm not sure why the closure of $\{y\}$ needs to be disjoint from $K \cup \{ x \}$ if the space isn't at least $T_1$.

ccaruvana commented:
I'm tempted to change T373 to "hemicompact + not compact -> Markov k-Rothberger" for the following reason, which relates to the @prabau 's comment relating to the issue of trivial covers.

Since we are adopting the notion of non-trivial open covers, if $X$ is compact, then the $k$-covers will be so that they only cover _proper_ compact subsets of $X$. Now, I'm not sure, but maybe the Fort space on the reals creates a problem here since the proof of "hemicompact -> k-Rothberger" would basically say to pick an open set $U$ so that $X \subseteq U$. But if we are excluding such open sets, such a move doesn't work. So the issue now is, is there a sequence of _proper_ compact subsets of the Fort space on the reals that does the same thing?

prabau commented:
To make things easier in the proof, one can assume wlog that the countable local base $(U_n)$ at $x$ has the $U_n$ strictly decreasing (if there is a minimal nbhd of $x$, that nbhd would be compact, which is contradict the assumption).

But I agree it's hard to conclude without $T_1$ that the closure of $\\{y\\}$ is disjoint from $K\cup\\{x\\}$.  At that level of generality, it would not even be true.  For example if $K$ is a single point $z\ne x$, it could be that $z\in\overline{\\{y\\}}$, so no nbhd of $z$ can avoid $y$.

When we added https://topology.pi-base.org/theorems/T000322, Steven and I had the same discussion in nearly the same context.  We could not figure out how to remove the $T_1$ assumption.  See the reference for that: https://math.stackexchange.com/questions/4585825.

prabau commented:
> Since we are adopting the notion of non-trivial open covers, if $X$ is compact, then the $k$-covers will be so that they only cover _proper_ compact subsets of $X$. Now, I'm not sure, but maybe the Fort space on the reals creates a problem here since the proof of "hemicompact -> k-Rothberger" would basically say to pick an open set $U$ so that $X \subseteq U$. But if we are excluding such open sets, such a move doesn't work. So the issue now is, is there a sequence of _proper_ compact subsets of the Fort space on the reals that does the same thing?

I don't think there is issue if the space $X$ is compact.  The definition of $k$-cover says it cannot contain $X$ and for every compact subset of $X$ blah blah.  Note that it does not say "for every proper compact subset of $X$".  So if $X$ is compact, no $k$-cover of $X$ exists.  That in turn implies that $X$ is vacuously $k$-Menger, $k$-Rothberger, Markov $k$-Rothberger, etc.  There is no need for any extra assumption in T373.

prabau commented:
About T374, I think it would be worth adding a mention of where the $T_1$ assumption is used.  Something like this maybe: "In the proof, the set $O_n(K)$ can be chosen equal to $X\setminus\\{x_n(K)\\}$, which is open by the $T_1$ property."

ccaruvana commented:
According to one of the comments at https://mathoverflow.net/questions/431292 and the linked proof at https://dantopology.wordpress.com/2020/02/18/the-space-of-irrational-numbers-is-not-menger/, the irrational numbers Space 28 is an example of a Lindelof space which is not Menger. I'll get to looking for a peer-reviewed reference, but would a link to https://dantopology.wordpress.com/2020/02/18/the-space-of-irrational-numbers-is-not-menger/ suffice?

StevenClontz commented:
We should link to Dan's blog, but here's a direct writeup as well: https://math.stackexchange.com/questions/4727296/is-every-lindelof-space-menger/4727297#4727297

StevenClontz commented:
> > What does the settings.json file do?
> 
> Actually, I was wondering that myself. It loaded up as soon as I opened the branch. I'm not sure if it has something do with how I was interfacing, but I'm sure it can be deleted.

Yeah if the github.dev settings get changed, they can be saved to that file. But at least for now we don't want that persisted for all owners of the repo so I've added it to `.gitignore` (though I should have done that separate to this PR, it's not worth fixing now).

StevenClontz reviewed (commented):
While I'm sure it's correct, the proof of spaces/S000022/properties/P000151.md feels like it could be aired out on Math.SE.

StevenClontz reviewed (commented):
`spaces/S000043/properties/P000149.md`

Shouldn't this be deducable? E.g. do we have that epsilon-space => Lindelof? If so, this should be deleted.

ccaruvana commented:
> `spaces/S000043/properties/P000149.md`
> 
> Shouldn't this be deducable? E.g. do we have that epsilon-space => Lindelof? If so, this should be deleted.

Well, we have that the Sorgenfrey Line S43 is *not* an epsilon-space, so I don't think Lindelof is deduced from there. Is it deduced from other properties already in place?

StevenClontz commented:
> > `spaces/S000043/properties/P000149.md`
> > Shouldn't this be deducable? E.g. do we have that epsilon-space => Lindelof? If so, this should be deleted.
> 
> Well, we have that the Sorgenfrey Line S43 is _not_ an epsilon-space, so I don't think Lindelof is deduced from there. Is it deduced from other properties already in place?

Ah never mind!

ccaruvana commented:
> While I'm sure it's correct, the proof of spaces/S000022/properties/P000151.md feels like it could be aired out on Math.SE.

What would be the best way to phrase that question? "Is every Fortissimo space strategically $\omega$-Menger?" Also, I recently answered one of my own questions immediately after posting and got a critical comment. I didn't get any downvotes, but it makes me less eager to repeat similar posts.

StevenClontz reviewed (commented):
`spaces/S000179/properties/P000026.md`

I feel like this has something to do with the fact that the space is not separable, and the finite powers of separable are separable. Is the finite product of metacompact, metacompact? Then https://topology.pi-base.org/theorems/T000024 shows metacompact and separable gives not just Lindelof, but all finite powers lindelof.

`theorems/T000376.md`

https://topology.pi-base.org/spaces?q=Second%20Countable%2B~%24k%24-Lindel%C3%B6f is empty, but compare this with https://topology.pi-base.org/theorems/T000061

StevenClontz commented:
> > While I'm sure it's correct, the proof of spaces/S000022/properties/P000151.md feels like it could be aired out on Math.SE.
> 
> What would be the best way to phrase that question? "Is every Fortissimo space strategically ω-Menger?" Also, I recently answered one of my own questions immediately after posting and got a critical comment. I didn't get any downvotes, but it makes me less eager to repeat similar posts.

That user needs to read https://math.stackexchange.com/help/self-answer - I've reminded folks of this myself.

In https://math.stackexchange.com/questions/4727296/is-every-lindelof-space-menger/4727297#4727297 I explicitly pointed out my motivation to self-document an answer, maybe that helped.

StevenClontz commented:
As for phrasing, maybe just "what selection principles hold for the Fortissimo space" or similar? That also opens the doors for a usser to suggest a stronger selection principle to improve on your answer.

StevenClontz reviewed (approved):
I left a few comments, but importantly everything appears to be correct, so the improvements sugggested could happen in a follow-up PR.

That said, this PR is quite large; it'd have been nice if it was split up into some more managable chunks.

ccaruvana commented:
> I left a few comments, but importantly everything appears to be correct, so the improvements sugggested could happen in a follow-up PR.
> 
> That said, this PR is quite large; it'd have been nice if it was split up into some more managable chunks.

I agree with that. I had originally planned on doing just the $\omega$-principles first. I'm not sure if I had tried to duplicate the PR branch to have a separate PR. Is that something we can do and still push to main? Namely, can I go into a PR branch, copy it, make edits there, and do a PR to main based on the copied branch?

StevenClontz commented:
You can make a PR on a branch, then make a second branch directly from it,
and make a PR on that. But you're committing the second PR to assume the
first is successful.

On Thu, Jun 29, 2023, 3:32 PM Chris ***@***.***> wrote:

> I left a few comments, but importantly everything appears to be correct,
> so the improvements sugggested could happen in a follow-up PR.
>
> That said, this PR is quite large; it'd have been nice if it was split up
> into some more managable chunks.
>
> I agree with that. I had originally planned on doing just the $\omega$-principles
> first. I'm not sure if I had tried to duplicate the PR branch to have a
> separate PR. Is that something we can do and still push to main? Namely,
> can I go into a PR branch, copy it, make edits there, and do a PR to main
> based on the copied branch?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pi-base/data/pull/329#issuecomment-1613774308>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAL4YUCCH2ZAWLISZNC4VQ3XNXQ6BANCNFSM6AAAAAAZAULVEE>
> .
> You are receiving this because your review was requested.Message ID:
> ***@***.***>
>

prabau commented:
> What would be the best way to phrase that question? "Is every Fortissimo space strategically ω-Menger?" Also, I recently answered one of my own questions immediately after posting and got a critical comment. I didn't get any downvotes, but it makes me less eager to repeat similar posts.

I have found that if you post the question, wait a day or two, then answer the question yourself, it goes down very well.

prabau commented:
> I feel like this has something to do with the fact that the space is not separable, and the finite powers of separable are separable. Is the finite product of metacompact, metacompact? Then https://topology.pi-base.org/theorems/T000024 shows metacompact and separable gives not just Lindelof, but all finite powers lindelof.

The product of two metacompact spaces need not be metacompact.  Example: the Sorgenfrey line and the Sorgenfrey plane:
- https://topology.pi-base.org/spaces/S000043/properties/P000031
- https://topology.pi-base.org/spaces/S000076/properties/P000031

ccaruvana commented:
> `theorems/T000376.md`
> 
> https://topology.pi-base.org/spaces?q=Second%20Countable%2B~%24k%24-Lindel%C3%B6f is empty, but compare this with https://topology.pi-base.org/theorems/T000061

Now that you mention, second countability clearly implies being $k$-Lindelof. Take your countable basis and close it under finite unions. I'll update T376 to "Second Countable => k-Lindelof" in the PR to address Steven's comments.

StevenClontz commented:
> > `theorems/T000376.md`
> > https://topology.pi-base.org/spaces?q=Second%20Countable%2B~%24k%24-Lindel%C3%B6f is empty, but compare this with https://topology.pi-base.org/theorems/T000061
> 
> Now that you mention, second countability clearly implies being k-Lindelof. Take your countable basis and close it under finite unions. I'll update T376 to "Second Countable => k-Lindelof" in the PR to address Steven's comments.

Beat you to it with https://github.com/pi-base/data/pull/338 though I let the mob do the actual math for me haha.

ccaruvana commented:
Was looking around for an example of a Hausdorff space to fit in here: https://topology.pi-base.org/spaces?q=%24%5Csigma%24-relatively-compact%2B~%24%5Csigma%24-compact
Example 5.8 in https://doi.org/10.14712/1213-7243.2015.201 claims to do the trick, but I think the proof is faulty. Since the topology on the space includes Euclidean open sets, the open cover of $[-n,n] \setminus \mathbb Q$ consisting of ${ (-\infty, q) : q < n, q \in \mathbb Q \}$ has no finite subcover. Am I missing something?
